avocado/utils/genio.py

```diff
@@ -10,7 +10,12 @@
 #
 # This code was inspired in the autotest project,
 # client/shared/utils.py
-# Authors: Martin J Bligh <mbligh@google.com>, Andy Whitcroft <apw@shadowen.org>
+# Authors: Martin J Bligh <mbligh@google.com>
+#          Andy Whitcroft <apw@shadowen.org>
+#
+# Modification History:
+# Rohan Deshpande <rohan_d@linux.ibm.com>
+# - Added read_line_with_matching_pattern()
```

Comment:
as it is small addition so i feel we can drop that and we can get git history what added

Reply:
Done